src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -292,32 +253,25 @@ public static void UnexpectedException_Throws_FaultException()
         Assert.True(false, "Expected FaultException<FaultDetail> exception, but no exception thrown.");
     }
 
-    [Fact]
+    [Theory]
```

Comment:
As mentioned elsewhere, Theory won't run in our NET Native test environment.  Unfortunate, but I recommend an explicit loop over the theory data for data driven tests or separate tests if they really are different scenarios.